File: GJScrollViewUsage/GJScrollViewUsage/View/CompareButton.m
//
//  CompareButton.m
//  BuyCarDictionary
//
//  Created by zgjun on 14-12-31.
//  Copyright (c) 2014年 chexun. All rights reserved.
//

#import "CompareButton.h"

@implementation CompareButton


- (CGRect)titleRectForContentRect:(CGRect)contentRect {
    return CGRectMake(0.3 * self.width, 0, 0.7 * self.width, self.height);
}
- (CGRect)imageRectForContentRect:(CGRect)contentRect {
    return CGRectMake(0, 0, 0.3 * self.width, self.height);
}
@end

File: GJScrollViewUsage_Swift/GJScrollViewUsage_Swift/View/CompareButton.swift
//
//  CompareButton.swift
//  GJScrollViewUsage_Swift
//
//  Created by zgjun on 15/9/22.
//  Copyright (c) 2015年 chexun. All rights reserved.
//

import UIKit

class CompareButton: UIButton {

    override func titleRectForContentRect(contentRect: CGRect) -> CGRect {
         return CGRectMake(0.3 * contentRect.size.width, 0, 0.7 * contentRect.size.width, contentRect.size.height);
    }
    override func imageRectForContentRect(contentRect: CGRect) -> CGRect {
         return CGRectMake(0, 0, 0.3 * contentRect.size.width, contentRect.size.height);
    }
}
